mocca-client/src/main/java/com/paypal/mocca/client/MoccaSerializer.java

```diff
@@ -448,6 +460,11 @@ private void writeResponseTypeSelectionSet(final ByteArrayOutputStream requestPa
             final Type listResponseType = getInnerType(cfResponseType, List.class).orElse(cfResponseType);
             final Type rawResponseType = getInnerType(listResponseType, Optional.class).orElse(listResponseType);
 
+            if(isEnum(rawResponseType)){
```

Comment:
```suggestion
            if (isEnum(rawResponseType)) {
```

Reply:
In my IDE the spacing is right. Not sure why its not showing here in PR.

Reply:
Fixed the formatting.